### PR TITLE
Improve description of h_maxima and h_minima functions (#4928)

### LINKED ISSUE #4928: morphology.extrema.h_maxima and h_minima: slightly confusing function descriptions
## Description

Consider this very short piece of code:

```python
testimg = np.array([0, 1, 2, 3, 4, 3.5, 4, 5, 4, 3, 2, 1, 0])
morphology.h_maxima(testimg, 1)
morphology.h_maxima(testimg, 2)
morphology.h_maxima(testimg, 3)
```

The results show that the element with value 5 is always picked up by this function, even though it is not a local maximum with height h for any h; rather it is the global maximum, so will always satisfy the condition tested for by this function.

The function description should therefore mention that the global maximum is also included.  The same goes for the h_minima function.

Best wishes,

Julian

## Version information

As at GitHub commit https://github.com/scikit-image/scikit-image/commit/9fcb6c9d9cadb8efa6426f29c94ab59928f41761

### PR DESCRIPTION
**Pull request by juliangilbey**
## Description

Closes #4928 

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/morphology/extrema.py [hunk 1]
```diff
@@ -197,9 +203,9 @@ def h_minima(image, h, selem=None):
     Returns
     -------
     h_min : ndarray
-       The minima of depth >= h. The resulting image is a binary image, where
-       pixels belonging to the selected minima take value 1, the other pixels
-       take value 0.
+       The local minima of height >= h and the global minima.
```
Comment (grlee77) [inline skimage/morphology/extrema.py:206]: height -> depth ?

File: skimage/morphology/extrema.py [hunk 2]
```diff
@@ -178,10 +181,13 @@ def h_minima(image, h, selem=None):
...
     A local minimum M of depth h is a local minimum for which
-    there is at least one path joining M with a deeper minimum on which the
-    maximal value is f(M) + h (i.e. the values along the path are not
-    increasing by more than h with respect to the minimum's value) and no
-    path for which the maximal value is smaller.
+    there is at least one path joining M with an equal or lower local minimum
+    on which the maximal value is f(M) + h (i.e. the values along the path
+    are not increasing by more than h with respect to the minimum's value)
+    and no path to an equal or lower local minimum for which the maximal
+    value is greater.
```
Comment (grlee77) [inline skimage/morphology/extrema.py:188]: should this still be "smaller" rather than "greater" at the end of the paragraph?

**Comment by juliangilbey** [comment]
Aww foo :-(  Thanks @grlee77 - I'm glad you have such sharp eyes!

Fixed these two now.

**Comment by sciunto** [comment]
@grlee77 Are you satisfied by this PR now?  CI failure is unrelated.

**Comment by grlee77** [comment]
Thanks @juliangilbey

**Pull request merged**